components/brave_wallet_ui/components/extension/sign-panel/cow_swap_order.tsx

```diff
@@ -0,0 +1,249 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+import styled from 'styled-components'
+
+// Types / contants
+import { BraveWallet } from '../../../constants/types'
+import { UNKNOWN_TOKEN_COINGECKO_ID } from '../../../common/constants/magics'
+
+// Utils
+import { getLocale } from '../../../../common/locale'
+import { findTokenByContractAddress } from '../../../utils/asset-utils'
+import { getAccountLabel, getAddressLabel } from '../../../utils/account-utils'
+
+// Styled components
+import {
+  NetworkText,
+  StyledWrapper,
+  TopRow,
+  ButtonRow,
+  HeaderTitle
+} from './style'
+import { WalletButton } from '../../shared/style'
+import { CreateNetworkIcon } from '../../shared/create-network-icon/index'
+
+// Components
+import { NavButton } from '../buttons/nav-button/index'
+import {
+  TransactionQueueSteps //
+} from '../confirm-transaction-panel/common/queue'
+import { Origin } from '../confirm-transaction-panel/common/origin'
+import { SwapBase } from '../swap'
+import { EthSignTypedData } from './common/eth_sign_typed_data'
+
+// Queries
+import {
+  useGetCombinedTokensListQuery //
+} from '../../../common/slices/api.slice.extra'
+import {
+  useGetAccountInfosRegistryQuery, //
+  useGetNetworkQuery
+} from '../../../common/slices/api.slice'
+
+// Hooks
+import { useAccountOrb, useAddressOrb } from '../../../common/hooks/use-orb'
+
+const makeUnknownToken = (
```

Comment:
how about?
```suggestion
const UNKOWN_TOKEN = (
```

Reply:
I think `makeUnknownToken` makes more sense since it is a function. I'd use `UNKNOWN_TOKEN` for a constant.